src/dmd/backend/util2.d

```diff
@@ -282,108 +282,137 @@ int binary(const char *p, size_t len, const char ** table, int high)
  * If c is a power of 2, return that power else -1.
  */
 
-int ispow2(uint64_t c)
+int ispow2(ulong c)
 {       int i;
 
         if (c == 0 || (c & (c - 1)))
             i = -1;
         else
             for (i = 0; c >>= 1; i++)
-                ;
+            { }
         return i;
 }
 
 /***************************
  */
 
-#define UTIL_PH 1
+enum UTIL_PH = true;
 
-#if _WIN32
-void *util_malloc(unsigned n,unsigned size)
+version (MEM_DEBUG)
+    enum MEM_DEBUG = false; //true;
+else
+    enum MEM_DEBUG = false;
+
+version (Win32)
```

Comment:
Should be `Windows`  as `_WIN32` is defined for x64 aswell.

Reply:
Thanks, I had forgotten about that.